docs/contributors_guide/maintenance/iso_release_procedures/Build_simp_core_RPM_and_Deploy.rst

```diff
@@ -0,0 +1,9 @@
+Build Signed simp-core RPM and Deploy to Download Server
+========================================================
+
+.. NOTE::
+
+   For ``simp-core``, the ``pkg:single`` path will need to be used.
```

Comment:
This block wasn't introduced with this request. The only change was to the "Include" line at the end referencing a file name that changed. If this documentation needs to be updated I feel it would be best as a separate ticket as it probably requires re-writes of a few separate files that would be out of the scope of this specific ticket/request.

Reply:
Ticket created for this suggestion: https://simp-project.atlassian.net/browse/SIMP-7714